### PR TITLE
quiche: update QUICHE tar ball

### PR DESCRIPTION
**Pull request by danzh2010**
Signed-off-by: Dan Zhang <danzh@google.com>

Risk Level: low
Testing: existing tests passed

### CODE REVIEW

File: bazel/repository_locations.bzl [hunk 1]
```diff
@@ -808,8 +808,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "QUICHE",
         project_desc = "QUICHE (QUIC, HTTP/2, Etc) is Google‘s implementation of QUIC and related protocols",
         project_url = "https://quiche.googlesource.com/quiche",
-        version = "5dd7a030209f9a6b5043bebd8ac3ee54f18d1d08",
-        sha256 = "306342cb35cb9d8baea079c7b924b0133c53cbf182b251655e589d3b4604dc41",
+        version = "29fb6fc501e4df5874129408b5474a91eaa9b7a0",
+        sha256 = "32e7b8ce4e91bb45b617bbf9721132f9218b86d9b3821ebfa32b1c7fc01aafae",
```
Comment (moderation) [inline bazel/repository_locations.bzl:812]: Please update `release_date = "2021-08-06",` based on the commit https://quiche.googlesource.com/quiche/+/29fb6fc501e4df5874129408b5474a91eaa9b7a0

**Review by moderation — CHANGES_REQUESTED** [review]

File: bazel/repository_locations.bzl (hunk 1 above)
Comment (danzh2010) [reply bazel/repository_locations.bzl:812]: done

**Review by ggreenway — APPROVED** [review]
LGTM

**Pull request merged**